presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -190,6 +193,8 @@ private static void failRequirement(String format, Object... args)
 
     private static void warnRequirement(String format, Object... args)
     {
-        System.err.println("WARNING: " + format(format, args));
+        AnsiConsole.err.println(ansi()
```

Comment:
does this work well with log files?

Reply:
Yes, it removes escape sequences if it detects it's not a TTY